src/qt/rpcconsole.cpp

```diff
@@ -1120,11 +1120,12 @@ void RPCConsole::updateDetailWidget()
     if (stats->nodeStats.m_bip152_highbandwidth_from) bip152_hb_settings += (bip152_hb_settings == "" ? "From" : "/From");
     if (bip152_hb_settings == "") bip152_hb_settings = "No";
     ui->peerHighBandwidth->setText(bip152_hb_settings);
-    ui->peerLastSend->setText(stats->nodeStats.nLastSend ? GUIUtil::formatDurationStr(GetSystemTimeInSeconds() - stats->nodeStats.nLastSend) : tr("never"));
-    ui->peerLastRecv->setText(stats->nodeStats.nLastRecv ? GUIUtil::formatDurationStr(GetSystemTimeInSeconds() - stats->nodeStats.nLastRecv) : tr("never"));
+    const int64_t time_now{GetSystemTimeInSeconds()};
+    ui->peerConnTime->setText(TimeDurationField(stats->nodeStats.nTimeConnected, time_now));
```

Comment:
This refactoring changes the logic of the `peerConnTime` value setting, as now "Never" is a possible option.
Does it makes sense? Is it guaranteed to be safe?

Reply:
Good point. I was seeing output of `0s` and not "Never" but no reason to change the logic. Changed.